### PR TITLE
Custom time zone support

### PR DESCRIPTION
**Pull request by KamiSempai**
There is a lot of use cases when time zone should different from time zone of device. This update allow to use custom time zone.

### CODE REVIEW

File: library/src/test/java/com/squareup/timessquare/CalendarPickerViewTest.java [hunk 1]
```diff
@@ -17,6 +19,7 @@
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
+import static android.R.attr.timeZone;
```
Comment (edenman) [inline library/src/test/java/com/squareup/timessquare/CalendarPickerViewTest.java]: unused?

File: library/src/main/java/com/squareup/timessquare/CalendarPickerView.java [hunk 2]
```diff
@@ -246,14 +259,69 @@ public FluentInitializer init(Date minDate, Date maxDate, Locale locale) {
...
+   * Both date parameters must be non-null and their {@link Date#getTime()} must not return 0. Time
+   * of day will be ignored.  For instance, if you pass in {@code minDate} as 11/16/2012 5:15pm and
+   * {@code maxDate} as 11/16/2013 4:30am, 11/16/2012 will be the first selectable date and
+   * 11/15/2013 will be the last selectable date ({@code maxDate} is exclusive).
+   * <p>
+   * This will implicitly set the {@link SelectionMode} to {@link SelectionMode#SINGLE}.  If you
+   * want a different selection mode, use {@link FluentInitializer#inMode(SelectionMode)} on the
+   * {@link FluentInitializer} this method returns.
+   * <p>
+   * The calendar will be constructed using the given time zone and the default locale as returned by
```
Comment (edenman) [inline library/src/main/java/com/squareup/timessquare/CalendarPickerView.java]: looks like checkstyle is unhappy with this line being too long

**Comment by edenman** [comment]
And one last thing: you'll have to sign the [Square CLA](https://docs.google.com/forms/d/13WR8m5uZ2nAkJH41k7GdVBXAAbzDk00vxtEYjd6Imzg/viewform?edit_requested=true) for me to merge+release this.  Please comment when it's done so I get notified.  

Thanks for the PR!

File: library/src/test/java/com/squareup/timessquare/CalendarPickerViewTest.java (hunk 1 above)
Comment (KamiSempai) [inline library/src/test/java/com/squareup/timessquare/CalendarPickerViewTest.java]: Ups. I forgot to trim imports. Yes, this import is unused.

**Comment by KamiSempai** [comment]
License signed.

**Comment by edenman** [comment]
Thanks again, @KamiSempai!  Just released this as 1.7.0, it should be live on maven central in a couple hours.

**Pull request merged**